### PR TITLE
Code Coverage: Org User Settings Service #3

### PR DESCRIPTION
**Pull request by jayfyle**
# Edit 1: Coverage at 100

org_user_100

### CODE REVIEW

File: src/app/core/services/org-user-settings.service.spec.ts [hunk 1]
```diff
@@ -55,6 +63,15 @@ describe('OrgUserSettingsService', () => {
     });
   });
 
+  it("post(): should send new user's settings", (done) => {
```
Comment (Dimple16) [inline src/app/core/services/org-user-settings.service.spec.ts:66]: use single quotes new org user settings

File: src/app/core/services/org-user-settings.service.spec.ts [hunk 2]
```diff
@@ -64,4 +81,84 @@ describe('OrgUserSettingsService', () => {
       done();
     });
   });
+
+  it('clearOrgUserSettings(): should clear org user settings', (done) => {
+    orgUserSettingsService.clearOrgUserSettings().subscribe((res) => {
+      expect(res).toEqual(null);
```
Comment (Dimple16) [inline src/app/core/services/org-user-settings.service.spec.ts:87]: `expect(res).toBeNull()`

**Review by Dimple16 — CHANGES_REQUESTED** [review]
added few comments

File: src/app/core/mock-data/org-user-settings.data.ts [hunk 3]
```diff
@@ -480,3 +480,163 @@ export const orgUserSettingsData3: OrgUserSettings = {
     allowed_payment_modes: [AccountType.PERSONAL, AccountType.CCC, AccountType.COMPANY],
   },
 };
+
+export const orgUserSettingsDataWoCCIDs: OrgUserSettings = {
```
[the remark this answers is not in the document]
Comment (jayfyle) [reply src/app/core/mock-data/org-user-settings.data.ts:484]: Yes

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**